app/javascript/chat/__tests__/message.test.jsx

```diff
@@ -11,12 +11,13 @@ const msg = {
   color: '#00FFFF',
 };
 
-const getMessage = (message) => (
+const getMessage = (message, props = {}) => (
```

Comment:
non-blocking: The default value is not necessary for the extra props. If it's `undefined` it will still spread to an empty object.

![image](https://user-images.githubusercontent.com/833231/93517609-b5de2080-f8f9-11ea-884f-48586db207cd.png)

```suggestion
const getMessage = (message, props) => (
```